site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -310,7 +310,11 @@ object TypelevelSitePlugin extends AutoPlugin {
       WorkflowJob(
         "site",
         "Generate Site",
-        scalas = List((ThisBuild / scalaVersion).value),
+        scalas = List(
+          githubWorkflowScalaVersions
+            .value
+            .headOption
+            .getOrElse((ThisBuild / scalaVersion).value)),
```

Comment:
This can be an empty `List(...)` now right?

Reply:
Oh, we'll have to clear the `sbtStepPreamble` as well.

Probably should be safe to make this similar kind of change for the publish job 🤔

Reply:
I wasn't sure if it mattered when generating sites (I haven't tried using the site plugin yet). I'll make it empty and we'll find out :)

Reply:
[Looks like it worked](https://github.com/typelevel/sbt-typelevel/actions/runs/4952791623/jobs/8859547230?pr=505)?